Index: app/src/main/java/kh/edu/rupp/ite/furniturestore/model/api/service/ApiService.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package kh.edu.rupp.ite.furniturestore.model.api.service\r\n\r\nimport kh.edu.rupp.ite.furniturestore.model.api.model.ApiResponse\r\nimport kh.edu.rupp.ite.furniturestore.model.api.model.CategoryModel\r\nimport kh.edu.rupp.ite.furniturestore.model.api.model.ProductList\r\nimport kh.edu.rupp.ite.furniturestore.model.api.model.Res\r\n\r\nimport kh.edu.rupp.ite.furniturestore.model.api.model.User\r\nimport retrofit2.Call\r\nimport retrofit2.http.GET\r\nimport retrofit2.http.Path\r\n\r\ninterface ApiService {\r\n    @GET(\"api/products\")\r\n    fun loadProductList(): Call<Res<ProductList>>\r\n    @GET(\"api/categories\")\r\n    fun loadCategories(): Call<CategoryModel>\r\n    @GET(\"api/categories_by_id/{id}\")\r\n    fun loadProductsByCategory(@Path(\"id\") id: Int): Call<ApiResponse>\r\n    fun loadProductList(): Call<List<ProductList>>\r\n\r\n    @GET(\"?name=luc\")\r\n    fun getUser(): Call<User>\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/kh/edu/rupp/ite/furniturestore/model/api/service/ApiService.kt b/app/src/main/java/kh/edu/rupp/ite/furniturestore/model/api/service/ApiService.kt
--- a/app/src/main/java/kh/edu/rupp/ite/furniturestore/model/api/service/ApiService.kt	(revision 7a6075847471b97c690c60249dc42d71b19d9512)
+++ b/app/src/main/java/kh/edu/rupp/ite/furniturestore/model/api/service/ApiService.kt	(date 1699687959793)
@@ -17,7 +17,6 @@
     fun loadCategories(): Call<CategoryModel>
     @GET("api/categories_by_id/{id}")
     fun loadProductsByCategory(@Path("id") id: Int): Call<ApiResponse>
-    fun loadProductList(): Call<List<ProductList>>
 
     @GET("?name=luc")
     fun getUser(): Call<User>
